src/focus-ring.js

```diff
@@ -1,67 +1,112 @@
-/* https://github.com/WICG/focusring */
+/* https://github.com/WICG/focus-ring */
 document.addEventListener('DOMContentLoaded', function() {
-    var hadKeyboardEvent = false,
-        keyboardModalityWhitelist = [ 'input:not([type])',
+    var hadKeyboardEvent = false;
+    var keyboardThrottleTimeoutID = 0;
+
+    var keyboardModalityWhitelist = [ 'input:not([type])',
```

Comment:
This could use a comment explaining why the whitelist exists. That it's for elements we want to _always_ display a `:focus-ring` for